lib/elixir/lib/system.ex

```diff
@@ -114,6 +114,11 @@ defmodule System do
     end
   end
 
+  # Returns OTP version that Elixir was compiled with.
+  defmacrop get_otp_version do
```

Comment:
Let's keep it simple for now. No need for the compiled variant unless people need it programatically - which we do for compiled endianness.